core/app/assets/js/layout/page.js

```diff
@@ -0,0 +1,58 @@
+import Header from './header';
+import Sidebar from './sidebar';
+import Content from './content';
+import Footer from './footer';
+
+export default function Page( props ) {
+	const getSideBar = () => (
+		<Sidebar>
+			{ props.sidebar }
+		</Sidebar>
+	),
+	getFooter = () => (
+		<Footer>
+			{ props.footer }
+		</Footer>
+	);
+
+	const uiTheme = elementorAppConfig.ui_theme;
+	let userPrefersTheme = '';
+
+	if ( 'auto' === uiTheme ) {
+		if ( window.matchMedia && window.matchMedia( '(prefers-color-scheme: dark)' ).matches ) {
+			userPrefersTheme = ' dark';
+		}
+	} else if ( 'dark' === uiTheme ) {
+		userPrefersTheme = ' dark';
+	}
+
+	return (
+		<div className={ props.className + userPrefersTheme }>
+			<div className="eps-app__lightbox">
+				<div className="eps-app">
+					<Header title={ props.title } buttons={ props.headerButtons } />
+					<div className="eps-app__main">
+						{ props.sidebar && getSideBar() }
+						<Content>
+							{ props.content }
+						</Content>
+					</div>
+					{ props.footer && getFooter() }
```

Comment:
Check the prop from within the function,

Reply:
done